src/components/complex/ChainModal/index.jsx

```diff
@@ -49,7 +49,7 @@ const StyledFaInfoCircle = styled(FaInfoCircle)`
 `
 
 const ChainModal = ({ show, onClose }) => {
-  const activeChainId = useChainId()
+  const activeChainId = useNetwork()
```

Comment:
Why this change?

Reply:
This is the correct wagmi hook to use to get connected chain.

Reply:
OK so rename activeChainId into activeNetwork as you are extracting the chain id via .chain.id

Reply:
done